main/main.go

```diff
@@ -4,8 +4,10 @@ import "fmt"
 
 func main() {
 	fmt.Println("hello world!")
+	fmt.Println("Razvan Farte was here")
```

Comment:
how did this ended up here? 😄

Reply:
don't know, it was already on master :D

Reply:
well, that's why rebase > merge when you want to get your latest changes 😆